container-azure-csharp/Program.cs

```diff
@@ -42,10 +42,11 @@
     var image = new Docker.Image("image", new()
     {
         ImageName = Pulumi.Output.Format($"{registry.LoginServer}/{imageName}:{imageTag}"),
-        Build = new Docker.DockerBuild {
+        Build = new Docker.Inputs.DockerBuildArgs {
```

Comment:
When I run this, I get:

```
MSBuild version 17.5.1+f6fdcf537 for .NET
  Determining projects to restore...
  Restored /Users/cnunciato/Projects/dev/monica-csharp-pr/monica-csharp-pr.csproj (in 1.27 sec).
/Users/cnunciato/Projects/dev/monica-csharp-pr/Program.cs(45,35): error CS0234: The type or namespace name 'DockerBuildArgs' does not exist in the namespace 'Pulumi.Docker.Inputs' (are you missing an assembly reference?) [/Users/cnunciato/Projects/dev/monica-csharp-pr/monica-csharp-pr.csproj]
/Users/cnunciato/Projects/dev/monica-csharp-pr/Program.cs(49,38): error CS0234: The type or namespace name 'RegistryArgs' does not exist in the namespace 'Pulumi.Docker.Inputs' (are you missing an assembly reference?) [/Users/cnunciato/Projects/dev/monica-csharp-pr/monica-csharp-pr.csproj]

Build FAILED.
```

I think this is because the `.csproj` file needs to be updated -- will see if that's the case.

Reply:
Yeah that seems to be it -- in `{PROJECT}.csproj`, just change that `3.*` to `4.*`.